### ISSUE TITLE
VS is unable to create the .vs-debugger directory

### ISSUE THREAD

**Issue by MV10** [MicrosoftDocs/visualstudio-docs#9336]
My documentation issue is to note that it would be nice if the docs could provide hints about how to troubleshoot remote debugging, and where to go to report issues. 

I have searched every Visual Studio-related repository on github, and I can't find anything that seems relevant.

Does anybody know where I _should_ report the actual problem I'm having, below?

I'm running the latest VS2022 on Win10 Pro x64 and attempting to debug a simple Linux ARM32 .NET 6.0.18 program running on a Raspberry Pi 4B. The account is good, all per-requisites are satisfied, the SSH connection works, it shows me the process list, but when I try to start debugging, it fails. The Output window shows it could not create the `.vs-debugger` directory (see below). If I manually create the directory ahead of time, it seems to attach, but nothing works -- breakpoints are ignored, for example.

```
Initialization log:
Determining user folder on remote system...
Checking for existing installation of debugging tools...
Downloading debugger launcher...

Creating debugger installation folder: /home/jon/.vs-debugger
Failed: The specified directory /home/jon/.vs-debugger could not be created or accessed.
Parameter name: path

Unable to find debugger script at '/home/jon/.vs-debugger'.
Failed: Unable to find debugger script at '/home/jon/.vs-debugger'.
The program '[2214] RemoteDebugTarget' has exited with code 4294967295 (0xffffffff).
```

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 2ecc1202-a940-caa0-968f-9abfe2679d8b
* Version Independent ID: 20ed4d8d-35ff-f1fe-74e4-ce2c27bb72c8
* Content: [Debug .NET Core on Linux - Visual Studio (Windows)](https://learn.microsoft.com/en-us/visualstudio/debugger/remote-debugging-dotnet-core-linux-with-ssh?view=vs-2022)
* Content Source: [docs/debugger/remote-debugging-dotnet-core-linux-with-ssh.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/main/docs/debugger/remote-debugging-dotnet-core-linux-with-ssh.md)
* Product: **visual-studio-windows**
* Technology: **vs-ide-debug**
* GitHub Login: @Mikejo5000
* Microsoft Alias: **mikejo**

**Comment by WilliamAntonRohm** [comment]
@MV10 -- Jon, thank you for your thoughtful feedback.  Please consider these resources, if you haven't already:
- Ask a question or find an answer on the [Microsoft Q&A](https://learn.microsoft.com/en-us/answers/tags/176/vs) page for Visual Studio.
- Check the [Visual Studio MSDN Forum](https://social.msdn.microsoft.com/Forums/vstudio). 
- [Stack Overflow](https://stackoverflow.com/search?q=%22visual+studio%22)  
- [Reddit](https://www.reddit.com/r/VisualStudio)

You are welcome to report this issue here:
- Report product issues to us via the [Report a Problem](https://docs.microsoft.com/visualstudio/ide/how-to-report-a-problem-with-visual-studio) tool that appears both in the Visual Studio Installer and in the Visual Studio IDE.​ 
- Suggest a feature, track product issues, and find answers in the [Visual Studio Developer Community](https://developercommunity.visualstudio.com).

@Mikejo5000 -- Mike, please look into the documentation aspects of this issue.

**Comment by MV10** [comment]
I say this in the spirit of honest feedback: frankly the MSDN forums are mostly cut-and-paste responses from what I'd consider level 1 support types, and the bizarre/terrible UI/UX is a complete turn-off. In decades of programming I'd be hard-pressed to think of a single useful answer I've ever seen there. I only click those search results out of extreme desperation.

I did report it through VS, but unfortunately that seems to be "where bug reports go to die".

https://developercommunity.visualstudio.com/t/VS2022-remote-debugging-over-SSH-does-no/10394545

I really wish MS would just bite the bullet and shift everything to Github issues. I know, not helping. Anyway, I appreciate the feedback and hope someone can think of some useful troubleshooting tips to add.

As a more broadly-scoped thought, it would be nice in general if all docs would link back to relevant repos when they reference anything that is either open-sourced or at least discussed here openly. Big ask, I know.

**Comment by MaximG1234** [comment]
Hi @MV10 , I ran into the exact same issue but just wanted to note that I was able to resolve it by just creating the .vs-debugger folder in my user path. Can confirm the app is hitting breakpoints in my case. 

Are you taking into account that the application will have already started so you will need some sort of sleep prior to attaching to the process to ensure that you are actually able to hit a breakpoint?

**Comment by MV10** [comment]
Yes, thanks for the thoughts but I covered that. If you look at the VS bug report linked above, I tried several scenarios. I even tried GDB, and while that can't debug managed code, it accurately reported live activity like library loading, and to my surprise breakpoints still worked.

**Comment by Mikejo5000** [comment]
Asking about this.

**Comment by Mikejo5000** [comment]
@MV10 This appears to be a bug rather than a doc issue. There is an internal bug tracking the Developer Community issue that you opened. Closing this one.

**Comment by MV10** [comment]
@Mikejo5000 

Ok, but line one:

> My documentation issue is to note that it would be nice if the docs could provide hints about how to troubleshoot remote debugging, and where to go to report issues.

**Comment by gustavocalheiros** [comment]
any news on this bug? I have latest version of VS 2022 and it is still there, though it's reported as being fixed....

**Comment by GlitchyPie** [comment]
Just adding my 2 cents in that I ran into this VS 2022 17.7.5 when trying to remote debug a linux build from windows.  VS comes up with an error dialog:
```
Unable to find debugger script at
'/home/<user>/.vs-debugger'
```
I admittedly should probably have looked deeper at the output windows to try and figure stuff out, but as has been pointed out there is barely a mention in the docs that this could be an issue.

Once I created the .vs-debugger directory all worked fine, including breakpoints etc.
